meson.build

```diff
@@ -19,6 +19,8 @@ conf.set_quoted('PACKAGE', meson.project_name())
 conf.set_quoted('VERSION', meson.project_version())
 
 conf.set('HAVE_STRNDUP', cc.has_function('strndup', prefix: '#define _GNU_SOURCE\n#include <string.h>'))
+conf.set('HAVE_STRCASECMP', cc.has_function('strcasecmp', prefix: '#define _GNU_SOURCE\n#include <string.h>'))
+conf.set('HAVE_STRICMP', cc.has_function('_stricmp', prefix: '#include <string.h>'))
```

Comment:
I suggest just checking for a MSVC macro at compile time instead of adding configure-time overhead. This seems overengineered when it's really just one compiler which doesn't support this; and having both function checks makes sense only if you really use both, but your code change doesn't have a case for both checks failing, so it just pretends to be complete.

Reply:
Alright, will do. I wasn't entirely sure about build environments like mingw or cross-compiling from Linux, since I don't use that. I'll use the `_MSC_VER` macro